maquetado/setup/index.html

```diff
@@ -0,0 +1,21 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML & CSS Training</title>
+    <meta name="description" content="training">
+    <meta name="author" content="Wolox">
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+    <link rel="stylesheet"  type="text/css" href="application.css">    
+  </head>
+  <body>
+    <div class="title">
+      <img src="img/triangle.svg"/>
+      <p>Front-End Tools</p>
```

Comment:
Try to replace both elements by a single one and add the other as a pseudo-element.

I think this one is in a big font size, would fit more using `h2` instead of `p`